server/routes/wazuh-utils.js

```diff
@@ -23,6 +23,15 @@ export function WazuhUtilsRoutes(server) {
     }
   });
 
+  // Returns the config.yml file in raw
+  server.route({
+    method: 'PUT',
+    path: '/utils/updateconfiguration',
```

Comment:
The path must be `/utils/configuration` because we have a GET route for the same file and this is a PUT route.